tests/modules/biobambam/bamsormadup/test.yml

```diff
@@ -0,0 +1,12 @@
+- name: biobambam bamsormadup test_biobambam_bamsormadup
+  command: nextflow run tests/modules/biobambam/bamsormadup -entry test_biobambam_bamsormadup -c tests/config/nextflow.config
+  tags:
+    - biobambam/bamsormadup
+    - biobambam
+  files:
+    - path: output/biobambam/test.bam
+      md5sum: 243a77fb0642fd46bb16a4d3432d19dc
+    - path: output/biobambam/test.metrics.txt
+      md5sum: 1721879bea1f3888ecd33b35e6ee0e72
+    - path: output/biobambam/versions.yml
+      md5sum: 841f96891b41daa43faf2072c08419f5
```

Comment:
```suggestion
```

We are starting to remove this as it makes automated version bumping of modules more difficult